x/gmm/keeper/msg_server_withdraw.go

```diff
@@ -0,0 +1,54 @@
+package keeper
+
+import (
+	"context"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/sideprotocol/side/x/gmm/types"
+)
+
+func (k msgServer) Withdraw(goCtx context.Context, msg *types.MsgWithdraw) (*types.MsgWithdrawResponse, error) {
```

Comment:
Not sure whether withdraw also need slippage check or not. it could be big difference in a extreme market

Reply:
in withdraw? we have slippage check? I'm not sure. slippage is required for only swap

Reply:
ok.